packages/svelte/src/internal/client/warnings.js

```diff
@@ -21,11 +21,12 @@ export function hydration_attribute_changed(attribute, html, value) {
 }
 
 /**
- * Hydration failed because the initial UI does not match what was rendered on the server
+ * Hydration failed because the initial UI does not match what was rendered on the server. The error occurred near %location%
+ * @param {string | undefined | null} [location]
  */
-export function hydration_mismatch() {
+export function hydration_mismatch(location) {
 	if (DEV) {
-		console.warn(`%c[svelte] ${"hydration_mismatch"}\n%c${"Hydration failed because the initial UI does not match what was rendered on the server"}`, bold, normal);
+		console.warn(`%c[svelte] ${"hydration_mismatch"}\n%c${`Hydration failed because the initial UI does not match what was rendered on the server. The error occurred near ${location}`}`, bold, normal);
```

Comment:
Looks like this means it would say "The error occurred near" when when no location is known. Does the "Check which of the alternatives to render" logic not work correctly for runtime warnings?

Reply:
Oh good catch — I think the logic is just faulty when the choice is between a message with no params and a message with one param, and we didn't catch it yet because all the other 'did you mean?'-type messages are a choice between one param and two. Investigating

Reply:
fixed. caught another case where this was failing